src/hooks/useScrollToBottom.ts

```diff
@@ -1,19 +1,18 @@
 import React, { useEffect } from "react";
 
 interface Props {
-  // action: object | [];
+  action: any;
   element: React.RefObject<HTMLDivElement>;
 }
 
 export const useScrollToBottom = (props: Props) => {
```

Comment:
Let's follow the most common convention for hooks. (deps goes as a second param)
```
export const useScrollToBottom = (element: React.RefObject<HTMLDivElement>, deps?: Array<any>) => {
```